static/locales/en-US.yaml

```diff
@@ -45,6 +45,7 @@ Global:
   Videos: Videos
   Shorts: Shorts
   Live: Live
+  Community: Community
```

Comment:
Please move the existing Community translations in all translation files, to avoid duplicate strings and to reduce the amount of work the translators have to do, the same thing was done in the original subscriptions tab pull request (#3725), which introduced the global object. You'll need to update the channel page to use the Global.Community string too. (Basically there is no point having global strings if we are only going to use them in one place)